lib/galaxy/webapps/galaxy/controllers/history.py

```diff
@@ -97,7 +97,12 @@ def display_by_username_and_slug(self, trans, username, slug, **kwargs):
         session = trans.sa_session
 
         user = session.scalars(select(model.User).filter_by(username=username).limit(1)).first()
-        history = session.scalars(select(model.History).filter_by(user=user, slug=slug, deleted=False).limit(1)).first()
+        history = session.scalars(
+            select(model.History)
+            .filter_by(user=user, slug=slug, deleted=False)
+            .order_by(model.History.importable.desc())
```

Comment:
would you mind adding a comment explaining why this order_by clause is essential? (i.e., expected to return the one importable slug if there are duplicate slug names)